scripts/index.js

```diff
@@ -30,11 +33,30 @@ function drawQuestionsToPage(grabQuestion) {
 
     questionBoxDiv.textContent= `Question: ${grabQuestion.question}`;
     questionDiv.append(questionBoxDiv);
-    console.log(answers)
-    console.log(shuffle(answers))
 
 }
 
+//the Fisher-Yates Shuffle function!
+function shuffle(array) {
+    var currentIndex = array.length, temporaryValue, randomIndex;
+  
+    // While there remain elements to shuffle...
+    while (0 !== currentIndex) {
+  
+      // Pick a remaining element...
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex -= 1;
+  
+      // And swap it with the current element.
+      temporaryValue = array[currentIndex];
+      array[currentIndex] = array[randomIndex];
+      array[randomIndex] = temporaryValue;
+    }
+    
```

Comment:
add a new variable that can hold the shuffled array

Reply:
define shuffledAnswers,
cross check with answers[3] for correct answer.
input into answers box 
all within shuffle function